tools/codegen/gen_functionalization_type.py

```diff
@@ -320,49 +392,42 @@ def emit_registration_helper(f: NativeFunction) -> Optional[str]:
             # because we don't want to decompose non-view ops that are composite, like `at::ones`.
             registration_str = f'static_cast<{sig.ptr_type()}>(at::native::{native_api_name})'
         else:
+            # non-composite view ops (and inplace ops) get a normal registration.
             registration_str = f'TORCH_FN(functionalization::{wrapper_name(f.func)})'
-
         return f'm.impl("{f.func.name}", {registration_str});'
 
-    return emit_registration_helper(f)
+    # Note: for now, this logic is meant to avoid registering functionalization kernels for mobile.
+    # At some point, Vulkan we'll want to use functionalization and we'll need to change this.
```

Comment:
Given where Edge is headed, it seems more likely that functionalization will be preapplied to the trace mobile wants to ship and they won't need these keys directly shipped

Reply:
yeah that makes sense - this comment was already here before the PR, but i'll remove it when I re-work the pass into a separate `Functionalize` and `AddBackViews` key in a later PR